components/pages/home/DailyQuote.js

```diff
@@ -1,15 +1,33 @@
-import styles from "./dailyquote.module.css";
+import styled from "styled-components";
+
+const StyledContainer = styled.div`
+  /*layout*/
```

Comment:
As far as I know will   `justify-content: center;  align-items: center;  text-align: center;` not have any effect here since the container is set to display block and not flex. Not sure though but maybe you can check again? 🙂